vmdb/app/models/ems_refresh/refreshers/openshift_refresher.rb

```diff
@@ -0,0 +1,14 @@
+module EmsRefresh
+  module Refreshers
+   class OpenshiftRefresher < BaseRefresher
+    include EmsRefresherMixin
+
+    def parse_inventory(ems, _targets = nil)
+      all_entities = ems.with_provider_connection(&:all_entities)
+      # all_entities.merge!(ems.kubernetes_connect.all_entities)
+      EmsRefresh.log_inv_debug_trace(all_entities, "inv_hash:")
```

Comment:
Does this need an `if DEBUG_TRACE` or does the method itself handle it (I can't remember)